macros/sql/dedupe.sql

```diff
@@ -0,0 +1,41 @@
+{%- macro dedupe(table, group_by, order_by=none) -%}
```

Comment:
We're leaning away from `table` as the name for arguments that take a `relation`, because people get confused about whether views are allowed for example. 
```suggestion
{%- macro dedupe(relation, group_by, order_by=none) -%}
```